pipeline/data_store.py

```diff
@@ -1346,58 +1346,58 @@ def get_zp(self, session=None, reload=False, provenance=None):
 
 
     def get_reference(self,
+                      search_by='image',
                       provenances=None,
-                      min_overlap=0.85,
-                      ignore_ra_dec=False,
-                      match_filter=True,
-                      match_target=False,
                       match_instrument=True,
-                      match_section=True,
+                      match_filter=True,
+                      min_overlap=0.85,
                       skip_bad=True,
                       reload=False,
+                      randomly_pick_if_mulitple=False,
```

Comment:
Hmm, OK.  I think I was thinking about putting an if block in the code below that issues a warning when there are multiple references, turning it into an error if randomly_pick_if_multiple were False.  But then I thought, no, I want to use the parameter multiple_ok.  And then I didn't implement either, and didn't clean up after myself.

I'm going to keep multiple_ok, because I think that's clearer, plus it's already documented, and I'm going to put in the if block.

Oh, wait... looking at the docs, multiple_ok had more implementation to do.  Which I will do.  This comes from before, when we just passed an ra/dec to image searches, to now when we can pass an area, and as such can limit what's returned by the overlap fraction.  Ideally, with a high enough overlap fraction, there will only be one reference for a given field.  But, maybe not.  So we need a parameter.

...in fact, there's more editing to be done, because I didn't update this code after Reference.get_references could handle areas.  So, I'm going to edit this function a whole bunch and ask you to review it again.